modules_for_testing/library_module.py

```diff
@@ -0,0 +1,107 @@
+"""LIBRARY MODULE"""
+
+
+# 1.  Библиотека¶
+
+#
+class Book:
+    """Books in the library are described in this class:
+    Title, Author, Pages, ISBN and current status:
+    is available, is booked (by whom), is taken (by whom)"""
+
+    def __init__(self, title, author, pages, isbn):
+        """When new book is added to the ibrary,
+        this method defines such parameters as:
+        title, author, pages and isbn."""
+
+        self.title = title
+        self.author = author
+        self.pages = pages
+        self.isbn = isbn
+
+        self.status = "Available"
+        self.reserved_by = None
+        self.taken_by = None
+
+
+class Reader:
+    """Readers that come to the library
+    are described in this class.
+    They can reserve, take and return books."""
+
+    def __init__(self, name):
+
+        self.name = name
+        # print(f"В библиотеку записался новый читатель: {self.name}")
+
+    def reserve_book(self, book):
+        """Reserving a book to a particular reader
+        prevents reserving/taking this book by other reader"""
+
+        if book.status == "Reserved" and book.reserved_by != self.name:
+            # print(f"Невозможно зарезервировать книгу '{book.title}' "
+            #       f"для {self.name} - она уже зарезервирована "
+            #       f"другим читателем!")
```

Comment:
Why did you decide to comment this code? This log will be visible in the test, but use logging module.

Reply:
Fixed